### ISSUE TITLE
[Translate] English version littered with {: target="_blank" rel="noopener noreferrer"} and some other issues

### ISSUE THREAD

**Issue by ignotus666** [jamulussoftware/jamuluswebsite#14]
Hi, I'm working on getting the Spanish version ready and have noticed that the English version is littered with this: "{: target="_blank" rel="noopener noreferrer"}" all over the place. Do I leave them there? Delete them?

Also, I know it has been done with the best of intentions, but I'm seeing a fair few additions in English that contain many mistakes, with some entries being close to unintelligible. E.g. there's a section in Client-Troubleshooting (Try to connect - Message in Jamulus Client on MacOSX) that is virtually unreadable and looks more like a kind of bug report - leaving aside linguistic issues I don't understand what it's purpose is in the wiki. Then there are syntax errors scattered here and there which will require some proofreading.

I'm also seeing that changes keep being made constantly to the files in English—my 2-day-old fork is already obsolete and I'll have to update all the files again to reflect those changes. I think there should be a "feature freeze" before encouraging translations because otherwise we just keep playing catch-up with an ever-moving target. I know there's a lot going on but I think we should use a more structured approach than the current scattergun one—e.g. bundle edits and commit them on a weekly or fortnightly basis after reviewing them carefully.

What do you guys think?

**Comment by ann0see** [comment]
I agree. I'd suggest to make a freeze in 2 days then we can start translating.

@gilgongo should have a look at that. 

I think we should also discuss the use of opening pages in a new tab. When do we need to do that? 

I put it mainly on links to external sites.

**Comment by gilgongo** [comment]
> I'm seeing a fair few additions in English that contain many mistakes, with some entries being close to unintelligible

@ignotus666 That makes two of us! When moving the content from the old wiki to the new site as I found various drive-by edits had been made that I either removed or re-wrote. There's no way of detecting changes on the Github wiki automatically and I didn't realise how random it had become. I'd not looked at the troubleshooting page - good spot! I'll have a look.

 > I think we should also discuss the use of opening pages in a new tab. 

Yes, we could adopt an "external site, new tab" convention perhaps. Not sure about pros/cons though. I think there can be accessibility issues with new tabs but I'm not sure.

**Comment by ann0see** [comment]
I have to revert a few commits. So before editing, please wait a few minutes.

**Comment by ann0see** [comment]
Ok. I was too late. But I'll add your changes later.

**Comment by ann0see** [comment]
Done. It's all back to normal now.

**Comment by gilgongo** [comment]
So content wise, shall we freeze changes to the English version for say, a month until 31st November? Translators can then get a bit more up to speed and we can work out a change control procedure in the mean time?

**Comment by ann0see** [comment]
I would first proofread and document possible changes to the Englisch version and then do a freeze. 

Meanwhile I think we should work out a plan what should be included in the wiki and what not. What’s the goal of the wiki in a whole? What is each site for? Who do we want to target with which site? Afterwards we can start translating.

**Comment by gilgongo** [comment]
OK we can proof them first.

As to the goal of the site overall, I'd say it's to explain what Jamulus is and how to use it. The first of those is relatively easy and I think we're probably doing it on the home page reasonably well but then that's always debatable. On that score, there have been ideas around "re-branding" Jamulus as a "family". So the project/brand overall is "Jamulus", the client is "Jamulus Music Room" the server is "Jamulus Server" and any utils or side projects follow the same naming convention a bit like Adobe do. This is to help comprehension since the word "Jamulus" on its own isn't very descriptive (although it's very useful for SEO). Seems like an interesting idea at least.

Otherwise, we don't have sales or donation drives, but the nearest we've got to that is our user survey, and of course new releases. So I guess we make announcements too (I sort of hacked the page footer for that in fact). We might call more attention to the forums perhaps. The issue of contribution is - I think - covered implicitly by the declaration of being open source. I think as long as we have a link to Github for that then anyone who wants to contribute will do so as the conventions for that are well established.

We could re-think the structure of the docs perhaps (I simplified the nav on that, and the Manual could maybe do with some re-formatting), but I'm not sure whether we'd just end up doing things differently rather than actually better. It's worth a think obviously. Personally, I think they're about 25% too verbose per screen and would like to make them less wordy. But that's debatable of course.

**Comment by ignotus666** [comment]
Unless anyone was thinking of adding more content I reckon we could leave it as "frozen" now - proofreading corrections aren't going to alter the translations or the content as such because translators aren't going to carry over syntactic or spelling mistakes into the translation.

I think I recall it being mentioned before, but I'd add a "Quick Start" guide - or modify the manual to turn it into one, and make that the place you land in via the "Getting Started" link. As it is now, the manual offers an overview of the gui elements but doesn't really give you a simple walk-through of what steps you actually need to take to get up and running - and the wiki can be quite daunting when you consider how many pages and how much text you have to trawl through to get to that point. My guess is that a lot of people would be able to get going with a short guide that listed the basic steps with a few screenshots, something like: Basic requirements (external sound card, wired Internet connection) -> Install application and drivers (links) -> launch client -> choose sound card, buffer size, audio quality, etc. in settings -> Set up profile -> Open server connection window -> Join server. Then mention that if you want detailed information about everything from setting up your own server to compiling it yourself and troubleshooting tips, there's the wiki. 

In short, the manual doesn't quite explain how to get up and running, and the wiki goes into too much detail - I think something in between is called for, for those who already know a bit about audio and computers or who just want to get it set up and explore it for themselves. Most people hardly ever read the full manual when they buy stuff (I most definitely don't) - they just want to turn the thing on and consult the instructions as and when they get stuck.

**Comment by gilgongo** [comment]
> I'd add a "Quick Start" guide - or modify the manual to turn it into one, and make that the place you land in via the "Getting Started" link

Hehe - that's actually exactly what's supposed to be happening today (home page start here > getting started). We just didn't call it "quick start" so as not to tempt fate :-) But I do agree we do have an odd divide between the "Manual" and what has been called the "wiki" in terms of documentation - you raise a good point about combining them. This was in fact originally intended but for reasons I won't bore you with (mostly as we were short for time in the face of huge numbers of people suddenly wanting to use Jamulus in April) we didn't.  

The very different installation and interface setups between OSes is a slight fly in the ointment for a "universal" quick start though, which is why the "getting started" page forks off to platform-specific instructions. But that's not a big deal if we can get people back into setup after that. 

There is also the unfortunate fact that perhaps unlike many other software programs, the diversity of elements out of our control means that most of our users will probably need quite specialist support at some point once they have passed through the initial setup. This has of course produced the somewhat heavy burden of the "wiki" to deal with this. But again I don't think this is huge issue. It just means we can't be as free and easy about "quick start" as we might and I think we also need to expose people to the reassurance that there in fact is a lot of support material should they need it later - which they probably will. And the "quicker" the start in terms of insulating people from the issues, the more likely they are to need help after. But these are for now just details I think.

**Comment by ann0see** [comment]
> As to the goal of the site overall, I'd say it's to explain what Jamulus is and how to use it.

Agree. Then this should be the main thing we focus on. So we will not explain how jamulus works technically? --> Should be debated.

>  I think they're about 25% too verbose per screen and would like to make them less wordy.
Yes. But we must think about what to remove. E.g. we could remove entries like "You can also compile jamulus..., also see this guide of..." This could be added only to the getting started page.

> As it is now, the manual offers an overview of the gui elements but doesn't really give you a simple walk-through of what steps you actually need to take to get up and running

Yep. I agree on that. We could remove this page (leave it in the main repo) and add a "What to do after install" page instead.

**Comment by gilgongo** [comment]
> We could remove this page (leave it in the main repo) and add a "What to do after install" page instead.

What's the reason we'd leave it in the main repo (I assume you mean [here](https://github.com/corrados/jamulus/blob/master/src/res/homepage/manual.md))?

**Comment by corrados** [comment]
> >  We could remove this page (leave it in the main repo) and add a "What to do after install" page instead.

> What's the reason we'd leave it in the main repo (I assume you mean here)?

You could move that page if you like.

**Comment by ann0see** [comment]
So a lot has changed in the last weeks ;-). I suggest once #58 is merged we should focus on a few important points:
1. clarify the contribution process #59 #60 #31 (also a clarification of who does what (project management). I‘m currently merging a lot of PRs of the translators but I don’t know if they are correct since I don’t speak all of these languages. Since gilgongo is the wiki manager who collects edits etc. I will not focus on that (so I will only help out rethinking structure of pages etc.). I can focus on the technical part (Jekyll, ...)) (in progress)
2. rethink the manual and make it more of a "do this and that after having installed Jamulus" guide  #103 (done, can be improved)
3. add breadcrumbs #9 (todo)
4. remove unnecessary target blank (todo)

**Comment by gegeweb** [comment]
> I think we should also discuss the use of opening pages in a new tab. When do we need to do that?

Good practice is to leave the choice to the reader. So don't put a target.

For external links, this is debatable.
But for internal links I asked myself the same question... to keep or not?

**Comment by ann0see** [comment]
> to get going with a short guide that listed the basic steps with a few screenshots, something like: Basic requirements (external sound card, wired Internet connection) -> Install application and drivers (links) -> launch client -> choose sound card, buffer size, audio quality, etc. in settings -> Set up profile -> Open server connection window -> Join server.


The onboarding page was basically the result of this. Buffer settings are still missing.

**Comment by ann0see** [comment]
Should this still be open?
